string link_name          # name of link
                          # link names are prefixed by model name, e.g. pr2::base_link
---
gazebo_msgs/LinkState link_state	#state of the link expressed in world NED frame
bool success              # return true if get info is successful
string status_message     # comments if available
